fundamentals/apple/gettingstarted/starter/Favourites/Shared/FavouriteNumberView.swift

```diff
@@ -0,0 +1,73 @@
+//
```

Comment:
nit: For the starter project, consider leaving placeholder comments where users should be writing code.

Reply:
It's literally what code completion gives me:

<img width="1152" alt="CleanShot 2022-10-28 at 00 33 25@2x" src="https://user-images.githubusercontent.com/232107/198410588-3bd2a543-580a-4a9f-9d17-67c7c3669362.png">

Also, the code in the function will indeed be called (verified using a breakpoint).